ctapipe/io/simteleventsource.py

```diff
@@ -878,9 +903,12 @@ def _fill_simulated_event_information(array_event):
         if mc_shower is None:
             return
 
+        # rounding errors might lead to values slightly above/below PI/2
```

Comment:
above/below +/- PI/2

Reply:
fixed in the docstring of the new method